web-components/src/components/modal/TxModal.tsx

```diff
@@ -1,4 +1,5 @@
 import React from 'react';
+import ReactHtmlParser from 'react-html-parser';
```

Comment:
I had to replace this dependency (which is no longer maintained) with `html-react-parser` during React 18 migration.
Could you also replace it in this PR to anticipate the migration merge?

Reply:
done